testng-core/src/test/java/test/dataprovider/issue2888/SkipDataProviderTest.java

```diff
@@ -0,0 +1,25 @@
+package test.dataprovider.issue2888;
+
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Listeners;
+import org.testng.annotations.Test;
+import test.SimpleBaseTest;
+
+@Listeners({SkipDataProviderListener.class})
+public class SkipDataProviderTest extends SimpleBaseTest {
```

Comment:
It is a sample, not a test.
Suffix it by Sample instead of Test and remove dependency to SimpleBaseTest.